examples/test_suite_1/test_suite_1.py

```diff
@@ -55,6 +55,11 @@ class SuiteTearDown(pykiso.RemoteTestSuiteTeardown):
     yaml configuration file)
     """
 
+    def tearDown(self):
```

Comment:
what does this tearDown add to the test, you are checking if True is True.

Reply:
I forgot to remove it ;)